lib/iris/tests/integration/netcdf/test_general.py

```diff
@@ -484,5 +484,43 @@ def test_path_string_save_same(self):
         self.assertCDL(tempfile_frompath)
 
 
+@tests.skip_data
+class TestWarningRepeats(tests.IrisTest):
+    def test_datum_once(self):
+        fnames = [
+            "false_east_north_merc.nc",
+            "non_unit_scale_factor_merc.nc",
+            "toa_brightness_temperature.nc",
+        ]
+        fpaths = [
+            tests.get_data_path(("NetCDF", "mercator", fname)) for fname in fnames
+        ]
+
+        with warnings.catch_warnings():
+            warnings.simplefilter("default")
+            with warnings.catch_warnings(record=True) as record:
+                for fpath in fpaths:
+                    iris.load(fpath)
+        assert len(record) == 1
+
+    def test_datum_default(self):
```

Comment:
Upon reflection, I think this test is less relevant to this PR and given that its behaviour might change in future it would be better to be removed.